tests/features/project/project_rsvp.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Custom RSVP form
```

Comment:
The implementation of this test is missing.

Reply:
I'll do the implementation in the next PR after I figure out the WebDriver.wait()

Reply:
Please remove this file and re-introduce in the relevant PR.